src/main/java/application/FormController.java

```diff
@@ -102,6 +111,7 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
         FormRecordProcessorHelper.processXML(formEntrySession.getSandbox(), formEntrySession.submitGetXml());
         SubmitResponseBean submitResponseBean = new SubmitResponseBean(formEntrySession);
         log.info("Submit response bean: " + submitResponseBean);
+        sessionRepo.delete(submitRequestBean.getSessionId());
```

Comment:
@benrudolph @snopoke  Actually now that I think about it fresh, we definitely shouldn't delete here - as currently configured, formplayer (like the old server) returns the submit XML to the browser which itself submits to HQ. There's potential for loss here so we'll definitely want to keep around on formplayer at this point.

Is there any reason that Formplayer shouldn't itself submit the XML to HQ? And then we can safely delete the session once that's confirmed. 

I have a TODO for this here - https://github.com/dimagi/commcare-hq/blob/def10b6b089b81048c3046cf7121f58897787091/corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js#L82-L82